scripts/installers/linux/install.sh

```
@@ -138,69 +138,51 @@ create_user() {
 
-    # Restore backups
-    if [[ -n "$env_backup" && -f "$env_backup" ]]; then
-        info "Restoring .env..."
-        cp "$env_backup" "${INSTALL_DIR}/.env"
-        rm "$env_backup"
+    info "Installing files to ${INSTALL_DIR}..."
+    if ! cp -a "${src_dir}/." "${INSTALL_DIR}/"; then
+        error "Copy failed, restoring the previous version..."
+        for d in $owned; do
+            if [[ -e "${INSTALL_DIR}/${d}.old" ]]; then
+                rm -rf "${INSTALL_DIR}/${d}"
```

Shellcheck (0.11.0)

[warning] Use "${var:?}" to ensure this never expands to / .

(SC2115)